### ISSUE TITLE
TypeError: (0 , _googleTranslateApiX.translate) is not a function

### ISSUE THREAD

**Issue by leons1767** [AidanWelch/google-translate-api#25]
The codes ran fine that last time I tested, about a month or so ago. And then, in my regression test today, it starts giving the above error. 

Does anyone have any idea?

**Comment by AidanWelch** [comment]
Hi, can you post the code?

**Comment by leons1767** [comment]
Hello @AidanWelch 

Specific to my environment, I'm using Expo SDK46 android development build to run the codes. I found that I upgraded the package version accidentally, and downgrading it to v10.4.2 fixes it. It broke somewhere after v10.4.2.  The latest version is broken as well. 
```
         await translate(trText, { to: lang})
             .then((value) => trResult.push({ language: lang, text: value.text, fromLang: value.from.language.iso}))
             .catch((error) => console.log('translation error :', error))
```
Thanks to look into it.

**Comment by AidanWelch** [comment]
@leons1767 

Can you check if the Fetch API is exposed in your environment?

**Comment by leons1767** [comment]
@AidanWelch 

Fetch is working fine in my environment.  I also have the below error with v10.5.4. 

Possible Unhandled Promise Rejection (id: 1):
TypeError: (0 , _googleTranslateApiX.default) is not a function

{arguments: ["Possible Unhandled Promise Rejection (id: 1):\nTypeError: (0 , _googleTranslateApiX.default) is not a function"], logger: console}

**Comment by AidanWelch** [comment]
@leons1767 

Can you try instead of doing

```js
const translate = require('google-translate-api-x');
// Or
import translate from 'google-translate-api-x';
```

Instead doing

```js
import { translate } from 'google-translate-api-x';
// or
const { translate } = require('google-translate-api-x');
```

I wonder if it's an issue with handling the default exports in that enviroment

**Comment by leons1767** [comment]
@AidanWelch 

I tried before I opened the issue, it was not working.

**Comment by AidanWelch** [comment]
@leons1767 Can you try installing this in your project and running in your babel config: https://babeljs.io/docs/babel-preset-env

That fixes the issue for me.

**Comment by leons1767** [comment]
Hi @AidanWelch 

I installed babel-preset-env, but what do I need to add in my babel.config.js? My current babel config in below.

```
module.exports = function(api) {
  api.cache(true);
  return {
    presets: ['babel-preset-expo'],
    env: {
      production: {
        plugins: ['react-native-paper/babel'],
      },
    },
    plugins: ['react-native-reanimated/plugin'],
  };
};
```

**Comment by AidanWelch** [comment]
After testing I actually ran into other issues with `babel-preset-env` so instead I started using `@babel/plugin-proposal-private-methods` which after installing you can add to your config like so:

```js
module.exports = function(api) {
  api.cache(true);
  return {
    presets: ['babel-preset-expo'],
    env: {
      production: {
        plugins: ['react-native-paper/babel'],
      },
    },
    plugins: ['react-native-reanimated/plugin', ['@babel/plugin-proposal-private-methods', { loose: true }]],
  };
};
```

**Comment by leons1767** [comment]
@AidanWelch 

Thanks for your time to support. I tried @babel/plugin-proposal-private-methods, added the exact babel config, and it's still giving the same error.

TypeError: (0 , _googleTranslateApiX.default) is not a function

**Comment by AidanWelch** [comment]
Hmm, I think it might be an issue with certain JS features not being supported in your version of expo or `babel-preset-expo` as it works perfectly fine with that config for me in expo 48.

**Comment by leons1767** [comment]
Unfortunately, I can't upgrade from Expo SDK46 to SDK48 yet, due to the dependency on RN0.71 in the SDK48. I guess I will have to do with v10.4.2 for now.

**Comment by AidanWelch** [comment]
Hmm, it may be possible to get it to work.  You could also try adding `@babel/plugin-proposal-object-rest-spread` like the last plugin.  And if that doesn't work there might be other babel config changes you can make to support newer JS features

**Comment by AidanWelch** [comment]
Should no longer be required in [v10.6.7](https://github.com/vitalets/google-translate-api/commit/70d75cd49ebee03056f141834b50519bb0e88d90)

**Comment by leons1767** [comment]
Hi @AidanWelch, 

I tried upgrading to v10.6.7 today. It is still the same error with Expo SDK 46.

**Comment by leons1767** [comment]
Just an update. I upgraded my app to Expo SDK 49, and the issue persists. The error is exactly the same. 

I noticed that you were using axios in v10.4.2, and changed to fetch after that. fetch is not a standard function in react native, and it requires to be imported `import {fetch} from 'react-native`.  Sorry that I didn't look into your codes until now. I accidentally discovered it while looking into all the dependencies that need to be upgraded for the Expo SDK upgrade.
